app/components/Documentation/EditDocumentation.tsx

```diff
@@ -142,6 +145,24 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
     });
   };
 
+  const onClickArticle = (id: string): void => {
+    if (
+      typeof setViewArticleDispatch === 'function' &&
+      typeof setEditArticleDispatch === 'function'
```

Comment:
Those variables can have other from function type? If you wanted to check it for `undefined` value, there's no need to check type of it

Reply:
It was Denis Romanov's decision. We can fix this during refactoring.